docker-build-env/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM ubuntu:focal
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-recommends install \
         ca-certificates \
         clang \
+        clang-format \
```

Comment:
Why do you add clang-format to the container, if clang-format isn't ran in the container?